front/app/component-library/utils/styleUtils.ts

```diff
@@ -258,6 +258,17 @@ export const stylingConsts = {
   border: `1px solid ${colors.divider}`,
 };
 
+export const getFontWeightCSS = (fontWeight: FontWeight) => {
+  switch (fontWeight) {
+    case 'bold': // Value of 700
+      return 'bold';
+    case 'medium':
+      return 600;
```

Comment:
Not to be a PITA, but one more recommendation. I read earlier that 600 is typically called "semi-bold". "medium" rather corresponds with 500.

https://developer.mozilla.org/en-US/docs/Web/CSS/@font-face/font-weight#common_weight_name_mapping

Anyway, if we would never use 500, there's no problem.

Reply:
I'll change it to "semi-bold" :+1: Wasn't aware of the above! Better to get it right from the beginning :) Thanks!